src/Control/Middleware/URLSpecialsMiddleware/FlushScheduler.php

```diff
@@ -28,15 +27,13 @@ trait FlushScheduler
      */
     public function scheduleFlush(HTTPRequest $request)
     {
-        $flush = array_key_exists('flush', $request->getVars() ?? []) || ($request->getURL() === 'dev/build');
-
-        $kernel = Injector::inst()->get(Kernel::class);
-        if (!$flush || (method_exists($kernel, 'isFlushed') && $kernel->isFlushed())) {
-            return false;
+        if ($request->getURL() === 'dev/build') {
```

Comment:
I've redone the logic on this one since it was already checked `Injector::inst()->get(Kernel::class)->isFlushed()` and returning false if `isFlushed()` was true.